prow/spyglass/spyglass.go

```diff
@@ -75,10 +83,11 @@ type ViewRequest struct {
 }
 
 // New constructs a Spyglass object from a JobAgent and a list of ArtifactFetchers
-func New(ja *jobs.JobAgent, c *storage.Client) *Spyglass {
+func New(ja *jobs.JobAgent, conf configAgent, c *storage.Client) *Spyglass {
```

Comment:
not totally related to this PR as I'm pretty sure we do this elsewhere in Prow already but - I really dislike public APIs taking private types without at least documenting what real value(s) they expect :(

Can we at least update the comment here?

Reply:
There are indeed a lot of these around prow. In at least once case it is actually exported, though.

Will update the comment.

Reply:
Updated!